SignInCheckIn/SignInCheckIn/Controllers/ChildCheckinController.cs

```diff
@@ -61,5 +62,52 @@ public IHttpActionResult CheckinChildrenForCurrentEventAndRoom(ParticipantDto pa
                 throw new HttpResponseException(apiError.HttpResponseMessage);
             }
         }
+
+        [HttpGet]
+        [ResponseType(typeof(ParticipantDto))]
+        [Route("checkin/events/{eventId}/child/{callNumber}/rooms/{roomId}")]
+        public IHttpActionResult GetEventParticipantByCallNumber(
+             [FromUri(Name = "eventId")] int eventId,
+             [FromUri(Name = "callNumber")] int callNumber,
+             [FromUri(Name = "roomId")] int roomId)
+        {
+            try
+            {
+                var child = _childCheckinService.GetEventParticipantByCallNumber(eventId, callNumber, roomId, true);
+                if (child != null)
+                {
+                    return Ok(child);
+                }
+                else
+                {
+                    return NotFound();
```

Comment:
https://www.lucidchart.com/techblog/2015/08/31/the-worst-mistake-of-computer-science/